pkgs/development/python-modules/blis/default.nix

```diff
@@ -51,17 +59,14 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "blis" ];
 
-  passthru = {
-    # Do not update to BLIS 0.9.x until the following issue is resolved:
-    # https://github.com/explosion/thinc/issues/771#issuecomment-1255825935
-    skipBulkUpdate = true;
-    updateScript = gitUpdater {
-      rev-prefix = "v";
-      ignoredVersions = "0\.9\..*";
-    };
```

Comment:
Why not change the `rev-prefix` to `release-`? Are you sure this is not working?

Reply:
It's working but shouldn't be necessary.

Reply:
> It's working but shouldn't be necessary.

You mean that `nix-update` works with the current git tags OK, even without this information? From my experience, it doesn't:

https://github.com/Mic92/nix-update/issues/9#issuecomment-1058215873

Reply:
You're right, I'll add it back.